System/Win32/Types.hsc

```diff
@@ -203,6 +207,21 @@ peekTStringLen = peekCStringLen
 newTString     = newCString
 -}
 
+-- | Check a 'FilePath' for internal NUL codepoints as these are
+-- disallowed in Windows filepaths. See #13660.
```

Comment:
which 13660 is this? GHC?

Reply:
https://gitlab.haskell.org/ghc/ghc/-/issues/13660